src/other/utils.jl

```diff
@@ -178,3 +178,155 @@ function _nt_like_hash(v, h::UInt)
 
     return xor(objectid(Tuple(propertynames(v))), h)
 end
+
+_findall(B) = findall(B)
+
+@inline _blsr(x)= x & (x-1)
+const _msk64 = ~UInt64(0)
```

Comment:
Is it really useful to define this constant? I would think using `~UInt64(0)` directly in the function is OK as the compiler should easily optimize it out.

Reply:
I also thought this, however, this is the way Julia Base does this (but yeah - let us move it)

Reply:
Ok, sure :)